attributes/default.rb

```diff
@@ -3,6 +3,7 @@
 include_attribute "hops"
 
 default['onlinefs']['version']                = "1.9-SNAPSHOT"
+default['onlinefs']['migrate']['versions']    = ""
```

Comment:
I don't think we need another parameter, we already have the list of all the versions here:
https://github.com/logicalclocks/conda-chef/blob/a971f776819b948da9799b1b238a8b9b7b219dd8/attributes/default.rb#L46

When running an upgrade, users need to specify the version they want to upgrade from:
https://github.com/logicalclocks/conda-chef/blob/a971f776819b948da9799b1b238a8b9b7b219dd8/attributes/default.rb#L40

So we can already figure out if they are upgrading from 3.5.0 or earlier and so we need to run the migration. I would also run the migration depending on the Hopsworks version, not the onlinefs version. This because the version of onlinefs might not change for several releases of Hopsworks. But we need to migrate only at the first Hopsworks upgrade.

I'm ok if we take migrations case by case, i.e., if you are upgrading from 3.5.0 or earlier, online-fs chef will run the consumer group migration. And not the same as we do in Hopsworks, since Hopsworks migrations should be more frequent than onlinefs migrations.

Reply:
Is it possible that the migration involve multiple version e.g from 3.4 to 3.7
```
# Current installed version
default["install"]["current_version"]             = "3.4.0"
# Update target
default["install"]["version"] = "3.7.0-SNAPSHOT"
```

Reply:
Yeah it's possible, but in this case we don't have to worry about. In this case, we only care from which version we are upgrading from. 
We only need to run this migration if the version is less or equal to 3.5.0.

The goal should be to minimize the number of attributes a user has to remember to set when doing upgrades. We already received complaints about having to set the `current_version` attribute, I don't want to introduce a new one.